core/trino-spi/src/test/java/io/trino/spi/predicate/TestDomain.java

```diff
@@ -287,7 +287,7 @@ public void testEquatableNotNull()
         assertTrue(domain.includesNullableValue(0L));
         assertFalse(domain.includesNullableValue(null));
         assertEquals(domain.complement(), Domain.onlyNull(ID));
-        assertEquals(domain.toString(), "[ EXCLUDES[  ] ]");
+        assertEquals(domain.toString(), "[ EquatableValueSet EXCLUDES[type=id, values=0, {}] ]");
```

Comment:
the "EXCLUDES" should be inside `EquatableValueSet[...]`, perhaps before `values=`